tests/test_maybe_importing.py

```diff
@@ -0,0 +1,33 @@
+from types import ModuleType
+
+import pytest
+from great_expectations.util import maybe_import_module, maybe_import_attr
+
+
+def test_maybe_import_installed_module():
+    module = maybe_import_module("json")
+    assert isinstance(module, ModuleType)
+
+
+def test_maybe_import_uninstalled_module():
+    module = maybe_import_module("this_module_does_not_exist")
+    with pytest.raises(ModuleNotFoundError):
+        module.something
+    assert "please install" in module.__dict__["gx_error_message"].lower()
+
+
+def test_maybe_import_existing_value_attr():
+    attr = maybe_import_attr("datetime", "datetime")
+    # This should not error
+    attr.now()
+
+
+def test_maybe_import_nonexisting_attr():
+    with pytest.raises(AttributeError):
```

Comment:
Can we verify that the error message gives helpful info?

Reply:
I had to change how I am doing these imports, see https://github.com/great-expectations/great_expectations/pull/7274#issuecomment-1452701350.

I verify the error message are as expected.